library/src/plugins/backend/watchers/patchElements.ts

```diff
@@ -501,9 +500,9 @@ function morphNode(
     for (const { name, value } of (newNode as Element).attributes) {
       if (
         (oldNode as Element).getAttribute(name) !== value &&
-        !preserveAttrs.includes(kebab(name))
```

Comment:
Not sure why we are applying kebab case to raw element attribute names here. Perhaps I shouldn't have changed this though because it's not related to my other changes.

Reply:
I took this change out. Originally I was trying to remove all the non attribute key uses of the case functions, but I ended up leaving them untouched and creating separate ones for case mods.